### ISSUE TITLE
[Wayland] Copying text appends null byte

### ISSUE THREAD

**Issue by Emik03** [Quaver/Quaver#4199]
### Describe the bug

Copying notes, copying text from ImGui editor plugins, or using `imgui.SetClipboardText` causes an unexpected null byte to be copied alongside the text. This causes dialogs that restrict character inputs to be unable to paste text copied within Quaver.

### To Reproduce

1. Launch the game on a machine that runs Wayland on Linux.
2. Open the editor.
3. Copy any hit object.
4. Attempt to paste it in the "Go To Objects" dialog. This will fail due to the appended null byte.

### Expected behavior

There is no null byte appended to the end of the string in all three scenarios. This extra character can also trip up plugins, as there is now a discrepancy between different operating systems:
```lua
local function isWayland()
    imgui.SetClipboardText("test")
    return imgui.GetClipboardText() ~= "test"
end
```

### Log Files

N/A

### Screenshots

![2024-06-27 01-35-18 remuxed](https://github.com/Quaver/Quaver/assets/14614115/769a433d-08a9-4496-beb4-32ae389b1c1a)

**Comment by Emik03** [comment]
Just to be clear, it's not a limitation with Wayland.
![image](https://github.com/Quaver/Quaver/assets/14614115/9a17b823-8b67-45a4-b916-9e1c5d6b4ba6)

**Comment by AiAe** [comment]
This issue has been fixed in 1.5.0
